dashboard/db/migrate/20240530170255_add_flagged_inappropriate_to_ai_tutor_interaction_feedbacks.rb

```diff
@@ -0,0 +1,6 @@
+class AddFlaggedInappropriateToAiTutorInteractionFeedbacks < ActiveRecord::Migration[6.1]
+  def change
+    add_column :ai_tutor_interaction_feedbacks, :inappropriate, :boolean
+    add_index :ai_tutor_interaction_feedbacks, :inappropriate
```

Comment:
What's the motivation for adding an index on this column?

Reply:
My thinking was that we'd want to query for all of the inappropriate ai tutor interaction feedbacks, but sounds like we're headed in a [different direction](https://codedotorg.slack.com/archives/C04VD4ZB6BT/p1717091636478109?thread_ts=1717090761.195869&cid=C04VD4ZB6BT). revisions incoming!